.vscode/settings.json

```diff
@@ -14,5 +14,6 @@
   "python.formatting.provider": "yapf",
   "python.venvPath": "${workspaceFolder}/.venv/",
   "python.pythonPath": "${workspaceFolder}/.venv/Scripts/python.exe",
-  "python.unitTest.pyTestEnabled": true
+  "python.unitTest.pyTestEnabled": true,
+  // "editor.formatOnSave": true,
```

Comment:
?

Reply:
I can delete this for now. This will autoformat on save, so we only commit formatted code. I didn't enable it here since there were already so many changes.